api/v1beta1/keystoneapi_webhook.go

```diff
@@ -167,3 +167,25 @@ func (r *KeystoneAPI) ValidateDelete() (admission.Warnings, error) {
 	// TODO(user): fill in your validation logic upon object deletion.
 	return nil, nil
 }
+
```

Comment:
also need to handle 0 ApiTimeout in webhook https://github.com/search?q=org%3Aopenstack-k8s-operators%20%22spec.APITimeout%20%3D%3D%200%22&type=code